assets/js/components/surveys/SurveyQuestionSingleSelect.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * SurveyQuestionSingleSelect component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../Button';
+import SurveyHeader from './SurveyHeader';
+import SurveyQuestionSingleSelectChoice from './SurveyQuestionSingleSelectChoice';
+
+// const MAXIMUM_CHARACTER_LIMIT = 100;
+
+/* eslint-disable camelcase */
+
+const SurveyQuestionSingleSelect = ( {
+	question,
+	choices,
+	answerQuestion,
+	dismissSurvey,
+} ) => {
+	const [ value, setValue ] = useState( '' );
+	const [ writeIn, setWriteIn ] = useState( '' );
+
+	const mappedChoices = choices.map( ( choice ) => ( {
+		...choice,
+		answer_ordinal: `${ choice.answer_ordinal }`,
```

Comment:
Why does this need to be cast as a string here? The ordinals are numbers by default.

Reply:
The `Radio` component requires value to be a string in its propTypes

https://github.com/google/site-kit-wp/blob/develop/assets/js/components/Radio.js#L92

Reply:
Fair enough. Part of me thinks we should be casting that in the component 😅 

Let's just leave a comment explaining why we do this at least, as it surprised me at first.

Reply:
Done :+1: